wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md

```diff
@@ -0,0 +1,369 @@
+# Featured artist showcase beatmaps
+
+Prior to the announcement of upcoming [featured artists](wiki/Featured_Artists), members of the [Mappers' Guild](/wiki/Mappers_Guild) who have achieved either Tier 1 (or higher) in [Guild rank](/wiki/Mappers_Guild#rewards) or been accepting certain [limited-time mystery quests](wiki/Mappers_Guild#quests) set by the Guild in advance are occasionally invited to create exclusive "showcase beatmaps" by using songs and other materials from osu!'s featured artist backlog that are yet to be released. Once it is time to publicly announce the featured artists in question, these beatmaps will be comprehensively covered in the corresponding [newsposts](https://osu.ppy.sh/home/news) and will occasionally be promoted on the [@osugame Twitter](https://twitter.com/osugame "Twitter") as well.
```

Comment:
```suggestion
Members of the [Mappers' Guild](/wiki/Mappers_Guild) occasionally create exclusive "showcase beatmaps" by using songs and other materials from osu!'s unreleased [featured artists](wiki/Featured_Artists). When a featured artist is publicly announced, these beatmaps are covered in the corresponding [newsposts](https://osu.ppy.sh/home/news) and are often promoted on the [@osugame Twitter](https://twitter.com/osugame).

To participate in featured artist showcase beatmaps, a member of the Mappers' Guild must accept a [limited-time mystery quests](wiki/Mappers_Guild#quests) or be directly invited to create a beatmap after reaching tier 1 (or higher) in [Guild rank](/wiki/Mappers_Guild#rewards).
```
no problem with the info, but the way the first sentence is worded is really hard to read (too many clauses)